google-cloud-bigquery/lib/google/cloud/bigquery/external/csv_source.rb

```diff
@@ -367,6 +367,70 @@ def skip_leading_rows= row_count
             @gapi.csv_options.skip_leading_rows = row_count
           end
 
+          ##
+          # The list of strings represented as SQL NULL value in a CSV file.
+          # null_marker and null_markers can't be set at the same time. If null_marker is
```

Comment:
I'm a bit confused by this discussion of `null_marker` and `null_markers`. There isn't a `null_marker` attribute on this (`CsvSource`) object. Or are you referring to something else? Can you clarify?